src/NUnitEngine/Addins/teamcity-event-listener/TeamCityEventListener.cs

```diff
@@ -31,10 +31,10 @@
 namespace NUnit.Engine.Listeners
 {
     // Note: Setting mimimum engine version in this case is
-    // purely documentary since engines prior to 3.6 do not
-    // check the EngineVersion property and will try to 
+    // purely documentary since engines prior to 3.4 do not
+    // check the EngineVersion property and will try to
     // load this extension anyway.
-    [Extension(Enabled = false, EngineVersion = "3.4.0.0")]
+    [Extension(Enabled = false, EngineVersion = "3.4")]
```

Comment:
This is one of the changes that I wasn't sure about. The changes on master and on the release were different. I am assuming 3.4 is correct because that matches the unit tests.